custom/aliases/example.aliases.sh

```diff
@@ -1,2 +1,9 @@
-# Add your own custom alias in the custom/aliases directory. Aliases placed
-# here will override ones with the same name in the main alias directory.
+# Add your own custom alias in the custom/aliases directory. Aliases placed here
```

Comment:
This line doesn't fit in 79 characters. Column 80 is reserved for the line continuation mark (such as `\`) in typical text editors.